images/airflow/2.9.2/python/mwaa/config/airflow.py

```diff
@@ -29,16 +29,23 @@ def _get_essential_airflow_celery_config() -> Dict[str, str]:
     """
     celery_config_module_path = "mwaa.config.celery.MWAA_CELERY_CONFIG"
 
-    return {
-        "AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__VISIBILITY_TIMEOUT": "43200",
-        "AIRFLOW__CELERY__BROKER_URL": get_sqs_endpoint(),
-        "AIRFLOW__CELERY__CELERY_CONFIG_OPTIONS": celery_config_module_path,
-        "AIRFLOW__CELERY__RESULT_BACKEND": f"db+{get_db_connection_string()}",
-        "AIRFLOW__CELERY__WORKER_ENABLE_REMOTE_CONTROL": "False",
-        # These two are not Celery configs per-se, but are used by the Celery executor.
-        "AIRFLOW__CORE__EXECUTOR": "CeleryExecutor",
-        "AIRFLOW__OPERATORS__DEFAULT_QUEUE": get_sqs_queue_name(),
-    }
+    executor_type = os.environ.get("MWAA__CORE__EXECUTOR_TYPE", "CeleryExecutor").lower()
```

Comment:
Defaulting to CeleryExecutor is now spread across 2 places.  We should ensure that only one default is set.